server/controllers/getTopics.js

```diff
@@ -0,0 +1,13 @@
+const { fetchTopics } = require("../models");
+
+getTopics = (req, res, next) => {
+  fetchTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
+    })
+    .catch((err) => {
```

Comment:
Do we need a catch block? Are we anticipating any errors with this endpoint? If not then we don't need it :)